tests/api2/test_435_smb_registry.py

```diff
@@ -157,39 +151,22 @@ def check_aux_param(param, share, expected, fruit_enable=False):
 
 
 @pytest.mark.parametrize('preset', PRESETS)
-def test__test_presets(setup_for_tests, share_presets, preset):
-    """
-    This test iterates through SMB share presets,
+def test__test_presets(setup_for_tests,  preset):
```

Comment:
flake8 abhors a vacuum...
```suggestion
def test__test_presets(setup_for_tests, preset):
```